src/System.IO.FileSystem/src/System/IO/FileStream.Unix.cs

```diff
@@ -45,7 +45,7 @@ private SafeFileHandle OpenHandle(FileMode mode, FileShare share, FileOptions op
                 _asyncState = new AsyncState();
 
             // Translate the arguments into arguments for an open call.
-            Interop.Sys.OpenFlags openFlags = PreOpenConfigurationFromOptions(mode, _access, options); // FileShare currently ignored
+            Interop.Sys.OpenFlags openFlags = PreOpenConfigurationFromOptions(mode, _access, share, options); // FileShare currently ignored
```

Comment:
@JeremyKuhne, @jkotas, is the FileStream code in corefx being used by anyone at this point?  Can we delete it all from corefx?